hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/utils/SynthTxnUtils.java

```diff
@@ -35,8 +34,10 @@
  * Some utilities related to synthetic transaction bodies.
  */
 public class SynthTxnUtils {
-    public static final Key IMMUTABILITY_SENTINEL_KEY =
-            Key.newBuilder().keyList(KeyList.DEFAULT).build();
+    // This key can no longer be an empty key list as it fails validation.  Use this key for now
+    public static final Key IMMUTABILITY_SENTINEL_KEY = Key.newBuilder()
```

Comment:
Is this for the same purpose as the `HOLLOW_ACCOUNT_KEY` seen earlier?

Do we have other keys like this?  Should there be a single place to put them?

Reply:
This is the only one I am aware of.